### PR TITLE
update some Rice references to clarify Golomb Rice

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: flac.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ For a user-oriented overview, see [About the FLAC Format](documentation_format_o
 # Acknowledgments
 
 FLAC owes much to the many people who have advanced the audio compression field so freely. For instance:
-- [A. J. Robinson](http://svr-www.eng.cam.ac.uk/~ajr/) for his work on [Shorten](http://svr-www.eng.cam.ac.uk/reports/abstracts/robinson_tr156.html); his paper is a good starting point on some of the basic methods used by FLAC. FLAC trivially extends and improves the fixed predictors, LPC coefficient quantization, and Rice coding used in Shorten.
+- [A. J. Robinson](http://svr-www.eng.cam.ac.uk/~ajr/) for his work on [Shorten](http://svr-www.eng.cam.ac.uk/reports/abstracts/robinson_tr156.html); his paper is a good starting point on some of the basic methods used by FLAC. FLAC trivially extends and improves the fixed predictors, LPC coefficient quantization, and Golomb Rice coding used in Shorten.
```
Comment (retokromer) [inline flac.md]: I personally prefer `Golomb-Rice coding` with a dash between the two names.

**Comment by bumblebritches57** [comment]
Oh shit, I actually just published a similar patchset in my fork

https://github.com/bumblebritches57/flac_markdown/tree/Rename-RICE-to-Exp-Golomb

**Comment by dericed** [comment]
no worries, I'll drop my PR in favor of your more comprehensive one

**Comment by bumblebritches57** [comment]
I'll go right ahead and create that PR right now.

**Pull request closed without merging**